api/V1/Recording.js

```diff
@@ -282,4 +282,102 @@ module.exports = (app, baseUrl) => {
         });
       }
     );
+
+  /**
+  * @api {delete} /api/v1/recordings/:id Delete an existing recording
+  * @apiName DeleteRecording
+  * @apiGroup Recordings
+  *
+  * @apiUse V1UserAuthorizationHeader
+  *
+  * @apiUse V1ResponseSuccess
+  * @apiUse V1ResponseError
+  */
+  app.delete(
+    apiUrl + '/:id',
+    passport.authenticate(['jwt'], { session: false }),
+    async (request, response) => {
+      log.info(request.method + " Request: " + request.url);
+
+      if (!requestUtil.isFromAUser(request))
+        return responseUtil.notFromAUser(response);
+
+      var id = parseInt(request.params.id);
+      if (!id)
+        return responseUtil.invalidDataId(response);
+
+      var deleted = await models.Recording.deleteOne(request.user, id);
+      if (deleted) {
+        responseUtil.send(response, {
+          statusCode: 200,
+          success: true,
+          messages: ["Deleted recording."],
+        });
+      } else {
+        responseUtil.send(response, {
+          statusCode: 400,
+          success: false,
+          messages: ["Failed to delete recording."],
+        });
+      }
+    }
+  );
+
+  /**
+  * @api {patch} /api/v1/recordings/:id Update an existing recording
+  * @apiName UpdateRecording
+  * @apiGroup Recordings
+  * @apiDescription This call is used for updating fields of current recordings.
+  * The only values that can be updated at the moment are the following ones.
+  * If any other values are requested to be updated the request will fail and no
+  * updates will occur.
+  * - location
+  * - comment
+  *
+  * @apiUse V1UserAuthorizationHeader
+  *
+  * @apiParam {JSON} updates Fields to update. If any keys are not reconised or can't be updated the whole request will fail and no updates will occur.
```

Comment:
There's probably no need to repeat the bit about the request failing if it's just been mentioned.

Reply:
Are the fields passed as a list? That could be clearer.

Reply:
It parses the JSON using the key as the column and value as the new value. How would you have a list organized? A list of tuples with [[key1, val1], [key2, val2]]?

Reply:
Ah of course, it's the fields and new values so a dict/object is more appropriate. `JSON` doens't automatically mean "object" by the way. It could be a JSON array.

Anyway, maybe change the documentation to say: "Object containing the fields to update and their new values." or something.

Reply:
Fixed.